src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -147,6 +147,45 @@ public function getDateTimeFormat()
         return $this->dateTimeFormat;
     }
 
+    /**
+     * @param \DateInterval $dateInterval
+     *
+     * @return string An ISO8601 duration
+     */
+    public static function getDuration(\DateInterval $dateInterval)
+    {
+        $datePartAttributes = [
+            'y' => 'Y',
+            'm' => 'M',
+            'd' => 'D',
+        ];
+        $timePartAttributes = [
```

Comment:
feel like this should be const, same as var before, @sonata-project/contributors wdyt?

Reply:
@OskarStark do I need to address this? There's been no response from @sonata-project/contributers

Reply:
If it does not need to change, please use a constant.

Reply:
Class Constants added